src/modules/Dashboard/features/RespondentData/RespondentDataSummary/Report/Charts/ScatterChart/ChartTooltip/ChartTooltip.tsx

```diff
@@ -55,11 +58,19 @@ export const ChartTooltip = ({ data, 'data-testid': dataTestid }: ChartTooltipPr
 
             <StyledFlexColumn>
               <StyledListItemButton
-                onClick={navigateToReviewAnswer}
+                onClick={() => navigateToReviewAnswer()}
```

Comment:
It looks tricky without some details when you add `() => navigateToReviewAnswer()` instead of `navigateToReviewAnswer`. Afterwards while refactoring in future someone may revert it.

Let's provide the explicit boolean property `isFeedbackVisible` = false
```suggestion
                onClick={() => navigateToReviewAnswer(false)}
```